config.js

```diff
@@ -61,12 +61,12 @@ module.exports = config = convict({
     channels: {
       doc: 'the channels to connect to',
       format: Array,
-      default: [ '##zuzakistan-lab' ]
+      default: [ '##dendodge' ]
```

Comment:
You can customise this per instance by creating a `production.json` (or development, or whatever):
```js
{
   "irc":{
      "channels":[
         "##dendodge"
      ]
   }
}
```
and then running with `NODE_ENV=production npm start`

Reply:
lol how did that get in there?